src/components/SideNav/SideNav.tsx

```diff
@@ -26,6 +26,28 @@ const SideNav = (props: SideNavProps): ReactElement => {
 		link.onClick();
 	};
 
+	const findLinkselected = () => {
+		let linkSelected: Link
+		links.forEach(link => link.forEach(l => l.label === defaultLink ? linkSelected = l : null))
```

Comment:
This is just a suggestion to make the code easier to read.

If we look at the "const links" in SideNav stories, you'll see it looks something like this
`const links = [
    [ //Section 1
        {...myLink1},
        {...myLink2},
    ],
    [ //Section 2
        {...myLink3},
        {...myLink4},
    ]
]
`

This means that the links prop has sections and each section has links, so I'd suggest doing something like this:
```suggestion
		links.forEach(section=> section.forEach(link => link.label === defaultLink ? linkSelected = link : null))
```